.percy.yml

```
@@ -0,0 +1,95 @@
+# Agent configuration
+agent:
+  # Asset discovery settings
+  asset-discovery:
+    # Network idle timeout for asset loading
+    network-idle-timeout: 2000
+    
+    # Request patterns to ignore
+    request-headers-to-record:
+      - authorization
+      - x-percy-token
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)